balancer_conn_wrappers.go

```diff
@@ -183,6 +184,9 @@ func (ccb *ccBalancerWrapper) handleResolvedAddrs(addrs []resolver.Address, err
 }
 
 func (ccb *ccBalancerWrapper) NewSubConn(addrs []resolver.Address, opts balancer.NewSubConnOptions) (balancer.SubConn, error) {
+	if len(addrs) <= 0 {
+		return nil, fmt.Errorf("grpc: cannot create SubConn with empty address list")
```

Comment:
Should we try to make `NewSubConn([])` do the same thing that `sc := NewSubConn([foo]); sc.UpdateAddresses([])` would do, for consistency?